.travis.yml

```diff
@@ -24,8 +24,13 @@ matrix:
 
     allow_failures:
         - php: hhvm
-        - env: SYMFONY_VERSION="^3.0.0"
-        - env: SYMFONY_VERSION="^3.0.0" COMPOSER_FLAGS="--prefer-lowest"
+        - env: SYMFONY_VERSION="dev-master"
```

Comment:
`SYMFONY_VERSION="dev-master"` is not in build matrix, so it does not make sense to allow failures for that.